Documentation/ceph-upgrade.md

```diff
@@ -228,9 +228,65 @@ The largest portion of the upgrade is triggered when the operator's image is upd
 kubectl -n $ROOK_SYSTEM_NAMESPACE set image deploy/rook-ceph-operator rook-ceph-operator=rook/ceph:v1.1.0
 ```
 
-Watch now in amazement as the Ceph mons, mgrs, OSDs, rbd-mirrors, mdses and rgws are terminated and
+#### 3.a Manually migrate override ConfigMap overrides
+If there are Ceph configuration overrides set in the `config` field of the ConfigMap
+`rook-config-override`, the Rook operator will fail before it begins orchestration of the Ceph
+cluster. The override ConfigMap is now deprecated in favor of a newly added `configOverrides`
+section of the `CephCluster` custom resource. It is important to wait until this initial operator
+failure, because it is only at this point that the `CephCluster` resource has been upgraded to the
+newest version by Rook. If the user has not added any overrides to the ConfigMap, this step will not
+be necessary.
+
+If the operator fails with an error about the legacy override ConfigMap, the user should now edit
+the `CephCluster` resource for their Ceph cluster and manually enter their overrides from the
+ConfigMap into the `configOverrides` section. A brief migration example is below.
+
+Ceph config file headers correspond to
+the `who` field of each config override as part of the section. For example, the `[global]` section
+corresponds to `who: global`, and `[osd.0]` corresponds to `who: osd.0`.
+
+```sh
+# List the contents of the override ConfigMap
+$> kubectl --namespace $ROOK_NAMESPACE describe configmap `rook-ceph-override` -o yaml
+Name:         rook-config-override
+Namespace:    rook-ceph
+Labels:       <none>
+Annotations:  <none>
+
+Data
+====
+config:
+----
+[global]
+debug_ms = 1/5
+
+[osd.0]
+debug_osd = 10
+Events:  <none>
+
+# Edit the Ceph cluster resource, and add the overrides from the ConfigMap
+$> kubectl --namespace $ROOK_NAMESPACE edit cephcluster $ROOK_NAMESPACE
+# ...
+spec:
+  # ...
+  configOverrides:
+  - who: global
+    option: debug_ms
+    value: 1/5
+  - who: osd.0
+    option: debug_osd
+    value: "10"
+    who: osd.0
+# ....
+```
+
+Once the migration is complete, restart the Rook Ceph operator by deleting the operator pod; it will
+be restarted automatically by the operator's deployment.
+
+#### 3.b Ceph cluster automatically updates
+Watch now in amazement as the Ceph mons, mgrs, OSDs, rbd-mirrors, MDSes and RGWs are terminated and
 replaced with updated versions in sequence. The cluster may be offline very briefly as mons update,
-and the Ceph Filesystem may fall offline a few times while the mdses are upgrading. This is normal.
+and the Ceph Filesystem may fall offline a few times while the MDSes are upgrading. This is normal.
```

Comment:
That shouldn't be the case since we do restart and wait before going to the next daemon but I see this is not coming from you, you simply renamed MDSes.